crates/rolldown/src/utils/chunk/render_chunk.rs

```diff
@@ -83,6 +83,11 @@ pub async fn render_chunk(
         }
       }
 
+      // TODO indent chunk content for iife format
```

Comment:
Does it mean we have to introduce `string_wizard`? If so, we need to rethink the importance of `string_wizard`.

Reply:
Yeah.